packages/modules/create-imodel/src/components/button-bar/ButtonBar.scss

```diff
@@ -0,0 +1,16 @@
+/*---------------------------------------------------------------------------------------------
+ * Copyright (c) Bentley Systems, Incorporated. All rights reserved.
+ * See LICENSE.md in the project root for license terms and full copyright notice.
+ *--------------------------------------------------------------------------------------------*/
+@import "~@itwin/itwinui-css/scss/variables";
+.iac-button-bar {
+    width: 100%;
+    margin-bottom: $iui-xl;
+    display: flex;
+    justify-content: center;
+    margin-top: $iui-m;
+  
+    > :first-child {
+      margin-right: $iui-m;
```

Comment:
instead, we can use `gap: $iui-m` on the parent element

Reply:
fixed